### ISSUE TITLE
Is this package still maintained, because it doesn't seem to work on android anymore

### ISSUE THREAD

**Issue by L0rdCr1s** [gre/react-native-view-shot#512]

**Comment by ErHarinderSingh** [comment]
Library looks dead now. ☹️

**Comment by ErHarinderSingh** [comment]
Package looks dead now. ☹️

**Comment by gre** [comment]
Hey there,

Thanks for flagging this. Let me provide some clarity. Yes, the package is still maintained, although it's now a community effort. While I'm not actively using it myself, I do review and merge PRs. However, recent issues on Android support might indeed be related to changes in React Native architecture.

🙏  we need someone with a keen understanding of these changes to dive into the issue and provide a fix. This is where your help could really make a difference. If you're up for it, please feel free to contribute a fix or even just lend your insights.

Looking forward to seeing what we can do together,
Thanks,
gre

**Comment by gre** [comment]
Yes! `v5@alpha` brings a full TypeScript rewrite, Fabric/new arch support, and RN 0.84+ compatibility. Stable v5 coming soon.

**Comment by gre** [comment]
Yes, alive and just shipped v5.0.0 with full New Architecture support (Fabric + TurboModules), RN 0.84 compatibility, TypeScript migration, and a long list of Android / iOS / web fixes. Please give it a try!

Release notes: https://github.com/gre/react-native-view-shot/releases/tag/v5.0.0
